pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -517,22 +500,21 @@ func TestSimpleDaemonSetScheduleDaemonSetPodsLaunchesPods(t *testing.T) {
 					pod.Name, len(nodeSelector.NodeSelectorTerms))
 			}
 
-			if len(nodeSelector.NodeSelectorTerms[0].MatchExpressions) != 1 {
-				t.Fatalf("incorrect expression number of pod %s node selector term, expected: 1, got: %d.",
-					pod.Name, len(nodeSelector.NodeSelectorTerms[0].MatchExpressions))
+			if len(nodeSelector.NodeSelectorTerms[0].MatchFields) != 1 {
+				t.Fatalf("incorrect Field number of pod %s node selector term, expected: 1, got: %d.",
```

Comment:
The grammar was weird before you touched it, so feel free to ignore this comment.

grammar nit: rather than "incorrect field number" (or incorrect expression number) it should read, "incorrect number of fields". Also, replace "of" with "in" or "for"

Reply:
Sure.